classes/class-mondu-api.php

```diff
@@ -0,0 +1,382 @@
+<?php
```

Comment:
Question: We have both tabs and spaces in this file for indentation. Which one is correct for wordpress marketplace? Lets only use 1 in our repository.